### PR TITLE
Show `View at this point in history` for every commit

### PR DESCRIPTION
**Pull request by zokkis**
Shows the 'View at this point in history'-link (from #27354) for every commit

before:
image

after:
image

### CODE REVIEW

File: templates/repo/commits_list.tmpl [hunk 1]
```diff
@@ -80,6 +80,8 @@
 							<button class="btn interact-bg gt-p-3" data-tooltip-content="{{ctx.Locale.Tr "copy_hash"}}" data-clipboard-text="{{.ID}}">{{svg "octicon-copy"}}</button>
 							{{if $.FileName}}
 								<a class="btn interact-bg gt-p-3" data-tooltip-content="{{ctx.Locale.Tr "repo.commits.view_path"}}" href="{{printf "%s/src/commit/%s/%s" $commitRepoLink (PathEscape .ID.String) (PathEscapeSegments $.FileName)}}">{{svg "octicon-file-code"}}</a>
+							{{else}}
+								<a class="btn interact-bg gt-p-3" data-tooltip-content="{{ctx.Locale.Tr "repo.commits.view_path"}}" href="{{printf "%s/src/commit/%s" $commitRepoLink (PathEscape .ID.String)}}">{{svg "octicon-file-code"}}</a>
```
Comment (delvh) [inline templates/repo/commits_list.tmpl]:
```suggestion
							<a class="btn interact-bg gt-p-3" data-tooltip-content="{{ctx.Locale.Tr "repo.commits.view_path"}}" href="{{if $.FileName}}{{printf "%s/src/commit/%s/%s" $commitRepoLink (PathEscape .ID.String) (PathEscapeSegments $.FileName)}}{{else}}{{printf "%s/src/commit/%s" $commitRepoLink (PathEscape .ID.String)}}{{end}}">{{svg "octicon-file-code"}}</a>
```

Comment (zokkis) [reply templates/repo/commits_list.tmpl]: I did it on multiple lines for better readability, but it should be ok also

**Review by delvh — APPROVED** [review]

**Review by KN4CK3R — APPROVED** [review]

**Comment by zokkis** [comment]
Do I have to 'Update branch' or something? Because it has 2 approvals but I cant merge

**Comment by KN4CK3R** [comment]
Users can't merge their PRs.

**Comment by wxiaoguang** [comment]
Actually I have a question for this change: if there is no `FileName`, why it should show a link to the commit? The commit link is already shown in the second column. If I understand correctly, `View at this point in history` is only for viewing a "file", but not a "commit". I think this change is not right.

**Comment by wxiaoguang** [comment]
-> Revert "Show View at this point in history for every commit (#29122)" #30295

**Pull request merged**